scripts/src/App/Command/GithubSync.php

```diff
@@ -0,0 +1,290 @@
+    ];
+    private const JIRA_LABELS = [
+        self::GH_LABEL_AutomationInProgress => 111,
+        self::GH_LABEL_Automated => 61,
+    ];
+
+    /** @var string */
+    protected $name = 'github:sync';
+
+    /** @var string */
+    protected $description = 'JIRA - Github : Sync';
+
+    /** @var Github */
+    private $github;
+
+    /** @var array */
+    private $issues;
+
+    /** @var boolean */
+    private $isVerbose = false;
+
+    /** @var OutputInterface */
+    private $output;
+
+    protected function configure()
+    {
+        parent::configure();
+
+        $this
+            ->addOption('ghtoken', null, InputOption::VALUE_OPTIONAL, '', getenv('GH_TOKEN') ?? null); 
+              
+    }
+ 
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        $time = time();
+
+        $this->apiKey = $input->getOption('apikey');
+        $this->requestsCount = 0;
+        $this->github = new Github($input->getOption('ghtoken'));
+        $this->isVerbose = $input->getOption('verbose');
+        $this->output = $output;
+
+        $this->issues = [];
+
+        // Fetch scenarios (To Be Automated / Automation In Progress / Automated)
+        $this->fetchJIRAScenarios();
+
+        foreach($this->issues as $key => $jiraIssue) {
+            $ghIssue = $this->getGithubIssue($key);
+            if (!$ghIssue) {
+                $this->actionCreateGHIssue($jiraIssue);
+                continue;
+            }
+
+            // Update body if different
+            $this->actionUpdateGHIssueBody($ghIssue, $jiraIssue);
+
+            // Update Github from JIRA (it there are no labels on GH)
+            $this->actionUpdateGHFromJIRA($ghIssue, $jiraIssue);
+
+            // Update JIRA from Github
+            $this->actionUpdateJIRAFromGH($ghIssue, $jiraIssue);
+        }
+        
+        $output->writeLn(['', 'Sync done in ' . (time() - $time) . 's with ' . $this->requestsCount . ' requests.']);
+    }
+
+    private function fetchJIRAScenarios(): void
+    {
+        foreach(['Core', 'Modules'] as $projectName) {
+            $folders = $this->getFolders($this->apiKey, $projectName);
+
+            $this->processJIRAFolder($folders, true);
+        }
+    }
+
+    private function processJIRAFolder(array $folder, bool $isRoot): void
+    {
+        // Process tests
+        $tests = $this->getTests($this->apiKey, $folder['id']);
+        foreach($tests as $test) {
+            if (!in_array(
+                $test['workflowStatus'],
+                [
+                    self::JIRA_ToBeAutomated,
+                    self::JIRA_AutomationInProgress,
+                    self::JIRA_Automated,
+                ]
+            )) {
+                continue;
+            }
+            $this->issues[$test['key']] = $test;
+            $this->issues[$test['key']]['url'] = 
+            'https://build.prestashop-project.org/test-scenarios/scenarios'
```

Comment:
maybe you can store this URL in a constant. It will be easier to change it if the project website will change / disappear one day.

Reply:
Fixed